mscore/startupWizard.cpp

```diff
@@ -0,0 +1,81 @@
+      }
+      Shortcut::loadFromNewFile(fileLocation);
+      }
+
+QMap<QString, QString> StartupWizard::getShortcutMap()
+      {
+      QMap<QString, QString> *layoutToShortcut = new QMap<QString, QString>();
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "US-QWERTY"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "UK-QWERTY"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "QWERTZ"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "AZERTY"), ":/data/shortcuts_AZERTY.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "BEPO"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "DVORAK"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "JIS - Standard Japanese"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "OYAYUBI SHIFUTO"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "T1 - Standard German"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "T2 - German"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Swiss German"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Belgian French"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Canadian French"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "French French"), ":/data/shortcuts_AZERTY.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Swiss French"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Spanish"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Italian"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Brazilian Portuguese ABNT"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Portuguese"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "US - International"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Dutch"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Korean - Dubeolsik"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Korean - Sebeolsik"), ":/data/shortcuts.xml");
+      layoutToShortcut->insert(QT_TRANSLATE_NOOP("keyboard-layout", "Other"), ":/data/shortcuts.xml");
+      return *layoutToShortcut;
+      delete layoutToShortcut;
```

Comment:
somewhere in the calling methods, after having used that QMap and picked an entry from it, I think